src/main/java/ladder/view/OutputView.java

```diff
@@ -61,4 +79,33 @@ private String generateRungMessage(boolean rungExist) {
         }
         return LADDER_RUNG_EMPTY;
     }
+
+    public void printPrizeNames(PrizesResponse prizesResponse) {
+        List<String> prizeNames = prizesResponse.prizeResponses().stream()
+                .map(PrizeResponse::name)
+                .toList();
+
+        String prizeNamesMessage = prizeNames.stream()
+                .map(this::generateNameMessage)
+                .collect(Collectors.joining());
+
+        System.out.println(prizeNamesMessage);
+    }
+
+    public void printPrizeForSelectedPlayer(PrizeResponse prizeResponse) {
+        String prizeName = prizeResponse.name();
+
+        System.out.println();
+        System.out.println("실행결과");
+        System.out.println(prizeName);
+    }
+
+    public void printAllResults(LadderAllResultsResponse ladderAllResultsResponse) {
+        Map<String, String> allResults = ladderAllResultsResponse.allResults();
+        String resultFormat = "%s : %s%n";
+
+        System.out.println();
+        System.out.println("실행결과");
+        allResults.forEach((playerName, prizeName) -> System.out.printf(resultFormat, playerName, prizeName));
```

Comment:
읽다가 람다 부분이 좀 헷갈려서 개행해줘도 좋겠네용
```suggestion
        allResults.forEach(
            (playerName, prizeName) -> System.out.printf(resultFormat, playerName, prizeName)
        );
```

Reply:
흠... 저는 이상하게 forEach에서 개행하면 어색하더라고요.

개행했습니다!

Reply:
혹시 프로그래밍 언어 어떤걸로 시작하셨어요? 
요런 감각은 출신 언어(??)에 따라 달라지기도 해서요, 감이 다를 땐 한번씩 여쭤봅니다 ㅎ ㅎ

Reply:
타입스크립트를 했었습니다! ㅎㅎ